modules/solid_mechanics/test/tests/rate_independent_cyclic_hardening/linear_kinharden_nonsymmetric_strain_controlled.i

```diff
@@ -0,0 +1,183 @@
+# This simulation uses the piece-wise strain hardening model
+# with the Finite strain formulation.
+#
+# This test applies a repeated displacement loading and unloading condition on
+# the top in the y direction. The material deforms elastically until the
+# loading reaches the initial yield point and then plastic deformation starts.
+#
+# The yield surface begins to translate as stress increases, but its size
+# remains the same. The backstress evolves with plastic strain to capture
+# this translation. Upon unloading, the stress reverses direction, and material
+# first behaves elastically. However, due to the translation of the yield surface
+# the yield point in the reverse direction is lower.
+#
+# If the reverse load is strong enough, the material will yield in the reverse
+# direction, which models the Bauschinger effect(reduction in yield stress in
+# the opposite direction).
+#
+# This test is based on the similar response obtained for a prescribed non symmetrical
+# stress path in Besson, Jacques, et al. Non-linear mechanics of materials. Vol. 167.
+# Springer Science & Business Media, 2009  pg. 89 fig. 3.6(a). This SolidMechanics code
+# matches the SolidMechanics solution.
+
+[Mesh]
+  file = 1x1x1cube.e
+[]
+
+[GlobalParams]
+  displacements = 'disp_x disp_y disp_z'
+[]
+
+[AuxVariables]
+  [total_strain_yy]
+    order = CONSTANT
+    family = MONOMIAL
+  []
+[]
+
+[Functions]
+  [top_pull]
+    type = PiecewiseLinear
+    xy_data = '0 0
+    0.025 0.0025
+    0.05 0.005
+    0.1 0.01
+    0.15 0.005
+    0.2 0
+    0.25 0.005
+    0.3 0.01
+    0.35 0.005
+    0.45 0
+    0.5 0.005
+    0.5 0.01
+    0.65 0.005
+    0.7 0
+    0.75 0.005
+    0.8 0.01
+    0.85 0.005
+    0.9 0
+    0.95 0.005
+    1 0.01
+    1.05 0.005
+    1.1 0
+    1.15 0.005
+    1.2 0.01
+    1.25 0.005
+    1.3 0
+    1.35 0.005
+    1.4 0.01
+    1.45 0.005
+    1.5 0'
+  []
+[]
+
+[Physics]
+  [SolidMechanics]
+    [QuasiStatic]
+      [all]
+        strain = SMALL
+        incremental = true
+        add_variables = true
+        generate_output = 'stress_yy plastic_strain_xx plastic_strain_yy plastic_strain_zz'
```

Comment:
can you create total_strain_yy here by doing this 
```suggestion
        generate_output = 'strain_yy stress_yy plastic_strain_xx plastic_strain_yy plastic_strain_zz'
```

Reply:
Yes, that is working. Removed Auxkernels and auxvariables blocks